cclib/parser/gamessparser.py

```diff
@@ -865,6 +864,8 @@ def extract(self, inputfile, line):
 
                 # Skip the reduced mass (not always present).
                 if line.find("REDUCED") >= 0:
+                    if not hasattr(self, "vibrmasses"):
+                        self.vibrmasses = []
```

Comment:
Reduced mass may not present in gamess output.
Add `vibrmasses` when it present.